internal/api/saml.go

```diff
@@ -89,7 +89,7 @@ func (a *API) SAMLMetadata(w http.ResponseWriter, r *http.Request) error {
 		var keyDescriptors []saml.KeyDescriptor
 
 		for _, kd := range spd.KeyDescriptors {
```

Comment:
/nit 

Consider updating the comment above to indicate that this is the default behaviour and tha

```
// don't advertize the encryption keys by default as it makes it much difficult to debug
	// requests / responses, and does not increase security since assertions are
	// not "private" and not necessary to be hidden from the browser
// You can opt out of this behaviour by enabling config.SAML.AllowEncryptedAssertions
```

Reply:
yea forgot this